buildSrc/src/main/groovy/wala-java.gradle

```diff
@@ -25,7 +25,7 @@ repositories {
 }
 
 // only use Error Prone on JDK 9+ JVMs
-final useErrorProne = JavaVersion.current().isJava9Compatible()
+project.ext.useErrorProne = JavaVersion.current().isJava9Compatible()
```

Comment:
@liblit is this the right way to do this?  I tried just doing `ext.useErrorProne = ...` but that did not make the variable visible inside `wala-NullAway.gradle`.

Reply:
Looks fine to me. I'm not sure why it didn't work without `project.`. But I'm also not very concerned about that: once we require Java 11+, this `useErrorProne` check can go away entirely.